src/main/java/io/vertx/core/eventbus/EventBus.java

```diff
@@ -77,10 +81,44 @@
    * @param options  delivery options
    * @param replyHandler  reply handler will be called when any reply from the recipient is received, may be {@code null}
    * @return a reference to this, so the API can be used fluently
+   * @deprecated use {@link #request(String, Object, DeliveryOptions, Handler)} instead
    */
   @Fluent
+  @Deprecated
   <T> EventBus send(String address, Object message, DeliveryOptions options, Handler<AsyncResult<Message<T>>> replyHandler);
 
+  /**
+   * Sends a message and and specify a {@code replyHandler} that will be called if the recipient
+   * subsequently replies to the message.
+   * <p>
+   * The message will be delivered to at most one of the handlers registered to the address.
+   *
+   * @param address  the address to send it to
+   * @param message  the message body, may be {@code null}
+   * @param replyHandler  reply handler will be called when any reply from the recipient is received
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  @SuppressWarnings("deprecation")
```

Comment:
Funny that it dispatches to the deprecated method 😉

Reply:
in master this will be different